apps/ejabberd/src/ejabberd_receiver.erl

```diff
@@ -142,27 +141,29 @@ init([Socket, SockMod, Shaper, MaxStanzaSize]) ->
 handle_call({starttls, TLSSocket}, _From, #state{parser = Parser} = State) ->
     NewParser = reset_parser(Parser),
     NewState = State#state{socket = TLSSocket,
-			   sock_mod = ejabberd_tls,
-			   parser = NewParser},
-    case ejabberd_tls:recv_data(TLSSocket, "") of
+         sock_mod = fast_tls,
```

Comment:
I think the indentation got broken here, the correct indentation should look like the following:

```erlang
NewState = State#state{socket = TLSSocket,
                                          sock_mod = fast_tls,
                                          parser = NewParser}.
```

With spaces of course :)
Please take a look at the comment "source code" as github renders the indentation incorrectly.